zone/groups.cpp

```diff
@@ -1140,12 +1140,12 @@ void Group::TeleportGroup(Mob* sender, uint32 zoneID, uint16 instance_id, float
 }
 
 bool Group::LearnMembers() {
-	//std::string query = StringFormat("SELECT name FROM group_id WHERE groupid = %lu", (unsigned long)GetID());
 	std::string query = StringFormat("SELECT name FROM group_id "
 		"WHERE group_id.groupid = %lu AND group_id.name NOT "
 		"IN(SELECT group_leaders.leadername FROM group_leaders WHERE gid = %lu)"
-		, (unsigned long)GetID()
-		, (unsigned long)GetID());
+		, GetID()
+		, GetID()
```

Comment:
Weird formatting.

Reply:
Thought I was following what I see in other places.  You may need to be more specific with me?

Reply:
The `,` should be on the previous line, not a new line in front of `GetID()` like it is here. `GetID(),` versus `, GetID()`.

Reply:
Got it.  Thanks.